mobile/src/reducers/settings.js

```diff
@@ -22,7 +22,7 @@ export const settings = (state = initialState, action) => {
     case INIT_STATE:
       return initialState
     case SET_CLIENT:
-      return { ...state, client: action.client, authorized: true }
+      return { ...state, authorized: true }
```

Comment:
I'm not sure because when I remove it, all application freeze... I can't debug. So I prefer no touch it ;)